### PR TITLE
Spinner: convert docs examples to Sandpack

### PR DESCRIPTION
**Pull request by dangerismycat**

### CODE REVIEW

File: docs/examples/spinner/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Fragment, type Node, useState } from 'react';
+import { Box, Button, Spinner } from 'gestalt';
+
+export default function Example(): Node {
+  const [show, setShow] = useState(true);
+
+  return (
+    <Fragment>
+      <Box padding={2}>
+        <Button
```
Comment (AlbertCarreras) [inline docs/examples/spinner/main.js:11]: Not sure if we need the Button hide/show functionality in the main example

**Review by AlbertCarreras — APPROVED** [review]

File: docs/pages/web/spinner.js [hunk 2]
```diff
@@ -56,35 +45,25 @@ function SpinnerExample() {
...
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description={`
+      Be sure to include \`accessibilityLabel\`. Labels should relate to the specific part of the product where Spinner is being used (e.g. "Loading homefeed" when used on the homefeed surface). Don't forget to localize the label!
+      `}
+      />
 
       <MainSection name="Variants">
-        <Example
+        <MainSection.Subsection
```
Comment (AlbertCarreras) [inline docs/pages/web/spinner.js:57]: Maybe we should add a variant "displayed" like the "paused" in Pulsar where we show the implementation.

File: docs/examples/spinner/main.js (hunk 1 above)
Comment (dangerismycat) [reply docs/examples/spinner/main.js:11]: As with the other: agreed that it's not a pattern we use much, but this is how the docs have been

File: docs/pages/web/spinner.js (hunk 2 above)
Comment (dangerismycat) [reply docs/pages/web/spinner.js:57]: Yeah, this page needs a lot more love. I'm going to leave as-is for now (the goal was just to convert the examples to Sandpack), but we should revisit

**Pull request merged**